src/app/utils/plug-descriptions.ts

```diff
@@ -85,29 +85,9 @@ export function usePlugDescriptions(
   const perks = getPerkDescriptions(plug, defs, statAndBungieDescStrings);
 
   if (showCommunityDescription && allClarityDescriptions) {
-    let clarityPerk = allClarityDescriptions[plug.hash];
-
-    // if we couldn't find a Clarity description for this perk, fall back to the non-enhanced perk variant
-    if (!clarityPerk) {
-      const regularPerkHash = enhancedPerkToRegularPerk[plug.hash];
-      if (regularPerkHash) {
-        clarityPerk = allClarityDescriptions[regularPerkHash];
-      }
-    }
-
-    if (clarityPerk && !clarityPerk.statOnly) {
-      // strip out any strings that are used in the Bungie description
-      const communityInsightWithoutBungieStrings = stripUsedStrings(
-        clarityPerk,
-        statAndBungieDescStrings
-      );
-      if (communityInsightWithoutBungieStrings) {
-        // if our stripped community description is truthy, we know it contains at least 1 unique string
-        // we only want to strip out Bungie description strings if we're also showing the Bungie description
-        result.communityInsight = showBungieDescription
-          ? communityInsightWithoutBungieStrings
-          : stripUsedStrings(clarityPerk, statStrings);
-      }
+    const clarityPerk = allClarityDescriptions[plug.hash];
+    if (clarityPerk) {
+      result.communityInsight = clarityPerk;
     }
```

Comment:
I'm OK with this logic being dropped if we're confident that Clarity descriptions will always provide more detail than the Bungie description.

If Clarity descriptions are occasionally going to be identical to the Bungie description (as was the case for **Invader Tracker**) or are simple stat strings that are already covered by the investment stats on the plug (e.g. `Grants 5 Stability` when there's already a `+5 Stability` investment stat present), then I would prefer that we keep this logic. In these scenarios, it makes sense to strip out duplicated text because they're redundant and are causing noise for the reader.

If we decide to keep this code, we'll need to update the `stripUsedStrings` function to use the currently selected language, to accommodate for Clarity descriptions being localised.

Reply:
GitHub isn't showing the diff properly - this is the removed code I'm referencing:
https://github.com/DestinyItemManager/DIM/blob/97c2792a5b37355e0a56714193a01240a9912d28/src/app/utils/plug-descriptions.ts#L98-L111

Reply:
Moved this logic to our editor if there are stats strings or Bungie descriptions its a bug on our and we will fix that

Reply:
Sounds good, thank you 👍